app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="left"
+    android:orientation="vertical"
+    tools:context=".SignUpActivity">
+    <!--TODO: replace with scroll view-->
+
+
+    <TextView
+        android:id="@+id/order_textview"
```

Comment:
As Moshe stated in class, let's stick with one convention and change all of these to the format of orderTextView, firstNameText, etc...

Reply:
@aviadud what do you think about it?

Reply:
I don't know how to do it.